python/pyspark/sql/connect/column.py

```diff
@@ -352,17 +365,307 @@ def __init__(self, expr: Expression) -> None:
     __rpow__ = _bin_op("pow", reverse=True)
     __ge__ = _bin_op(">=")
     __le__ = _bin_op("<=")
-    # __eq__ = _bin_op("==")  # ignore [assignment]
+
+    _eqNullSafe_doc = """
+        Equality test that is safe for null values.
+        .. versionadded:: 2.3.0
+        Parameters
+        ----------
+        other
+            a value or :class:`Column`
```

Comment:
```suggestion
            a value or :class:`Column`

```

Otherwise, the documentation format is broken